aws-transfer-connector/aws-transfer-connector.json

```diff
@@ -80,6 +80,7 @@
             "AES128_CBC",
             "AES192_CBC",
             "AES256_CBC",
+            "DES_EDE3_CBC",
```

Comment:
From @Alex-Vol-Amz's work to generate schemas from Smithy, I learned that new properties should be added to the end. I think this also applies to new enum values.

Reply:
Ahh I see,  pushed an update that has the right ordering